README.md

```diff
@@ -33,6 +33,26 @@ Install using `pip`:
 $ pip install uvicorn
 ```
 
+This will install uvicorn with minimal (pure Python) dependencies.
+
+```shell
+$ pip install uvicorn[standard]
+```
+
+This will install uvicorn with "fast" dependencies (where possible) and other "goodies".
+
+Fast meaning in that context that:
```

Comment:
Minor comment, but I'd probably phrase this slightly differently:

`In this context, "fast" means the following:` or

`"Fast" here means the following:`